pkg/sparrow/controller_test.go

```diff
@@ -302,45 +300,13 @@ func TestChecksController_UnregisterCheck(t *testing.T) {
 			check:   health.NewCheck(),
 			wantErr: false,
```

Comment:
This field can be removed now

Reply:
I'll do this in the #123, as the tests need to be refactored to truly test some use cases.